ansys/dpf/core/server_types.py

```diff
@@ -691,7 +691,7 @@ def ip(self):
         try:
             return self.info["server_ip"]
         except:
-            return 0
+            return ""
 
     @property
     def port(self):
```

Comment:
BTW @cbellot000 The `GrpcServer.external_ip` and `GrpcServer.external_port` docstrings mention `LegacyGrpcServer.ip` and `LegacyGrpcServer.port` instead of `GrpcServer.ip` and `GrpcServer.port`.

Reply:
Thanks @PProfizi I made another PR to fix that